packages/modules/vault/src/vault-container.test.ts

```diff
@@ -0,0 +1,54 @@
+import vault from "node-vault";
+import { setTimeout } from "node:timers/promises";
+import { StartedVaultContainer, VaultContainer } from "./vault-container";
+
+const VAULT_TOKEN = "my-root-token";
+
+describe("vault", { timeout: 180_000 }, () => {
+  let container: StartedVaultContainer;
+
+  afterEach(async () => {
+    await container?.stop();
+  });
+
+  // inside_block:readWrite {
+  it("should start Vault and allow reading/writing secrets", async () => {
+    container = await new VaultContainer().withVaultToken(VAULT_TOKEN).start();
+
+    const client = vault({
+      apiVersion: "v1",
+      endpoint: container.getAddress(),
+      token: container.getRootToken(),
+    });
+
+    await client.write("secret/data/hello", {
+      data: {
+        message: "world",
+        other: "vault",
+      },
+    });
+
+    await setTimeout(1000);
```

Comment:
Can be removed

Reply:
https://github.com/testcontainers/testcontainers-node/pull/1079/commits/94037565a65b84f5c45ee03fab16aac2ab4f2217